packages/auth/src/accessToken/service.ts

```diff
@@ -42,7 +48,8 @@ export async function createAccessTokenService({
   return {
     getByManagementId: (managementId: string) =>
       getByManagementId(managementId),
-    introspect: (tokenValue: string) => introspect(deps, tokenValue),
+    introspect: (tokenValue: string, access?: AccessItem[]) =>
```

Comment:
will/should `access` ever be undefined?

Reply:
It's optional in the GNAP spec. It's not required in the OpenAPI spec for the `token-introspection` package, but we certainly _could_ make it required. Payments are pretty sensitive entities, I'd be in favor of making it required.

Reply:
Edit, I think if we are doing [this](https://github.com/interledger/rafiki/pull/2788#discussion_r1690159011), ie returning the single relevant access item, we should make this required